packages/core/src/components/account/orders/MyAccountOrderDetails/MyAccountOrderDetails.tsx

```diff
@@ -48,12 +44,13 @@ export default function MyAccountOrderDetails({
             type="button"
             onClick={handleBack}
           />
-          <div data-fs-order-details-header-title-wrapper>
-            <h1 data-fs-order-details-header-title-text>
-              Order #{order.orderId}
-            </h1>
-            <UIBadge variant="warning">Pending approval</UIBadge>
-          </div>
+          <h1 data-fs-order-details-header-title-text>
+            Order #{order.orderId}
+          </h1>
+          <MyAccountStatusBadge
+            status={order.status}
+            statusFallback={order.statusDescription}
+          />
```

Comment:
O wrapper deve ter saído no merge
```suggestion
        <div data-fs-order-details-header-title-wrapper>
          <h1 data-fs-order-details-header-title-text>
            Order #{order.orderId}
          </h1>
          <MyAccountStatusBadge
            status={order.status}
            statusFallback={order.statusDescription}
          />
        </div>
```